### PR TITLE
Expand API support, including some V2 endpoints

### PR DESCRIPTION
**Pull request by tiagosiebler**
Add support for V2 versions of some APIs:
- place order
- cancel order

Add missing APIs:
- cancel all orders
- replace order
- get symbols

Add support for custom order ID for cancel active order.

### CODE REVIEW

File: lib/rest-client.js [hunk 1]
```diff
@@ -22,17 +22,52 @@ module.exports = class RestClient {
     return await this.request.post('/open-api/order/create', params);
   }
 
+  async placeActiveOrderV2(params) {
```
Comment (pixtron) [inline lib/rest-client.js]: I'd suggest to replace `placeActiveOrder` method with this one. The [the docs](https://github.com/bybit-exchange/bybit-official-api-docs/blob/master/en/rest_api.md#active-order) recommend to use v2 method over v1.

I would then publish a new minor or even major version of the module.

File: lib/rest-client.js [hunk 2]
```diff
@@ -22,17 +22,52 @@ module.exports = class RestClient {
...
 
   async cancelActiveOrder(params) {
     assert(params, 'No params passed');
-    assert(params.order_id, 'Parameter order_id is required');
+    assert(params.order_id || params.order_link_id, 'Parameter order_id OR order_link_id is required');
 
     return await this.request.post('/open-api/order/cancel', params);
   }
 
+  async cancelActiveOrderV2(params) {
```
Comment (pixtron) [inline lib/rest-client.js]: I'd suggest to replace `cancelActiveOrder` with this one, as the docs recommend to use v2.

**Review by pixtron — CHANGES_REQUESTED** [review]
Can you change `lib/rest-client.js`according to my comments below, and please add the new/changed methods to `doc/rest-client.md`.

**Comment by tiagosiebler** [comment]
Updated

**Comment by tiagosiebler** [comment]
@pixtron let me know if you need other changes to this, thanks!

**Comment by pixtron** [comment]
I will add some more missing api endpoints, which where introduced/documented after the release of this wrapper before releasing a new version.

**Comment by pixtron** [comment]
@tiagosiebler just published v1.1.0 including your changes.

**Pull request merged**